packages/platforms/browser/tests/page-load-span-plugin.test.ts

```diff
@@ -302,6 +305,39 @@ describe('FullPageLoadPlugin', () => {
     expect(delivery).toHaveSentSpan(expect.objectContaining({ name: '[FullPageLoad]/initial-route' }))
   })
 
+  it('starts a page load span when configured and updates the route on settle', async () => {
```

Comment:
A better test might be to start a new span and assert it has a parent span ID, then when the full page load span ends assert its span ID matches the parent span ID

Reply:
yeah good point. Also just realised that we're not setting `parentContext: null` when starting the page load span span, so if a span is already open the page load span will become a child (which is wrong).

Probably makes sense to add that as part of this PR

Reply:
Updated